doctrine.rst

```diff
@@ -479,7 +479,7 @@ for you automatically! First, install the bundle in case you don't have it:
 
 .. code-block:: terminal
 
-    $ composer require annotations
+    $ composer require doctrine/annotations
```

Comment:
That's not the same, `annotations` is an alias for the SensioFrameworkExtraBundle.